monitoring/replication/alerts.test.yaml

```diff
@@ -0,0 +1,455 @@
+        exp_alerts:
+          - exp_annotations:
+              description: Replication error rate is higher than 0.1%
+              summary: High ratio of replication errors
+            exp_labels:
+              severity: warning
+      - alertname: ReplicationErrorsCritical
+        eval_time: 10m
+        exp_alerts: []
+
+      - alertname: ReplicationErrorsWarning
+        eval_time: 14m
+        exp_alerts:
+          - exp_annotations:
+              description: Replication error rate is higher than 0.1%
+              summary: High ratio of replication errors
+            exp_labels:
+              severity: warning
+      - alertname: ReplicationErrorsCritical
+        eval_time: 14m
+        exp_alerts: []
+
+      - alertname: ReplicationErrorsWarning
+        eval_time: 15m
+        exp_alerts:
+          - exp_annotations:
+              description: Replication error rate is higher than 0.1%
+              summary: High ratio of replication errors
+            exp_labels:
+              severity: warning
+      - alertname: ReplicationErrorsCritical
+        eval_time: 15m
+        exp_alerts:
+          - exp_annotations:
+              description: Replication error rate is higher than 1%
+              summary: Very high ratio of replication errors
+            exp_labels:
+              severity: critical
+
+  - name: Rpo
+    interval: 1m
+    input_series:
+      # Requests last 50s for 5 min, then up to 600s for 5 min, then 900s for 5min
+      - series: replication_rpo_seconds_sum{namespace="zenko", job="backbeat-replication-data-processor-headless"}
+        values: 500+500x4 3000+6000x4 33000+9000x6
+      - series: replication_rpo_seconds_count{namespace="zenko", job="backbeat-replication-data-processor-headless"}
+        values: 10+10x16
+    alert_rule_test:
+      - alertname: RpoWarning
+        eval_time: 6m
+        exp_alerts: []
+      - alertname: RpoCritical
+        eval_time: 6m
+        exp_alerts: []
+
+      - alertname: RpoWarning
+        eval_time: 11m
+        exp_alerts:
+          - exp_annotations:
+              description: |
+                RPO is higher than 600 seconds.
+
+                This may indicate an issue with Kafka or the data-processor.
+
+                RPO is the delay between the time the object is updated and when it is picked up by the
+                data-processor.
+              summary: High replication RPO
+            exp_labels:
+              severity: warning
+      - alertname: RpoCritical
+        eval_time: 11m
+        exp_alerts: []
+
+      - alertname: RpoWarning
+        eval_time: 15m
+        exp_alerts:
+          - exp_annotations:
+              description: |
+                RPO is higher than 600 seconds.
+
+                This may indicate an issue with Kafka or the data-processor.
+
+                RPO is the delay between the time the object is updated and when it is picked up by the
+                data-processor.
```

Comment:
nit: not sure if it is supported by promtool, but we may want to use yaml anchors to avoid duplication if possible